templates/k-bims/0/docker-compose.yml

```diff
@@ -47,14 +47,17 @@ services:
     restart: unless-stopped
 
   web:
-    image: dimasciput/healthyrivers_nginx:latest
+    image: dimasciput/kartoza_nginx:latest
```

Comment:
Can we rather move this into Kartoza domain (give a specific version to the image) and then let all our Rancher configs move to this.

Reply:
I guess for develop branch using latest is fine - when we merge to master we should be using a specific version

Reply:
Github Actions supports semantic version tag releases.
It could be a free solution for this to build and push images with a tagged version that can be implemented in the cicd for this project, most cloud platform providers provide solutions for this as well ( mostly Terraform)... I don't know the context so making a CICD on my side will be difficult. 

There are also conflicts on the PR